README.md

```diff
@@ -6,6 +6,88 @@ Todo: Add table of contents
 
 # FitnessProject
 
+Getting Started:
+
+# MySQL setup for Mac 
+
+1. Download the MySQL DMG installer https://dev.mysql.com/downloads/mysql/
+2. Open the DMG file and follow the installation instructions.
+3. Open Terminal and edit your .zshrc (or .bash_profile if you're using Bash): nano ~/.zshrc
+4. Add the following line at the end of the file: export PATH="/usr/local/mysql/bin:$PATH"
+5. Save and close the file
+6. In Terminal, connect to the MySQL server: mysql -u root -p
+7. Enter your password when prompted.
+
+# MySQL setup for Windows TODO
+
+Case-insensitive table names
```

Comment:
## Install MySQL on Windows

**Note:** These instructions will guide you through the installation using Chocolatey. We recommended using Chocolatey for installing MySQL on Windows as it automatically handles dependency installation. 

### Part 1: Install Chocolatey
1. **Open an Administrative Command Prompt:**
   - Press `Win + R` to open the Run dialog.
   - Type `cmd`, then press `Ctrl + Shift + Enter` to open the command prompt with administrative privileges.

2. **Execute the Chocolatey Installation Command:**
   - Copy and paste the following command into the Command Prompt:
     ```bash
     Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
     ```
   - Press `Enter` to run the command and install Chocolatey on your system.

### Part 2: Install MySQL
1. **Open an Administrative Command Prompt** (repeat the process from Step 1 if necessary).

2. **Install MySQL Using Chocolatey:**
   - Type the following command:
     ```bash
     choco install mysql
     ```
   - Press `Enter`. Chocolatey will manage the download and installation of MySQL along with its dependencies.

### Part 3: Start MySQL Server
- **Start the MySQL Service:**
  - Type the following command in the Command Prompt:
    ```bash
    net start MySQL
    ```

### Troubleshooting Common Errors
1. **Access Denied Error:**
   - If you encounter the following error:
     ```
     ERROR 1045 (28000): Access denied for user 'ODBC'@'localhost' (using password: NO)
     ```
   - Log in using:
     ```bash
     mysql -u username -p  # if your MySQL user has a password
     ```
     or
     ```bash
     mysql -u username     # if your MySQL user does not have a password
     ```

2. **MySQL Service Not Starting:**
   - **Error Message:**
     ```
     Windows could not start the MySQL service on Local Computer. Error 1067: The process terminated unexpectedly.
     ```
   - **Check the MySQL Error Log:**
     - Locate the `my.ini` file in your MySQL server directory (typically under `C:\Program Files\MySQL\MySQL Server X.X`).
     - Find the `log-error` entry to locate your error log file path.
     - Navigate to the directory containing the error log and open the file (often named something like `hostname.err`) in a text editor.

   - **Verify MySQL Data Directory:**
     - Open your MySQL configuration file (`my.ini` or `my.cnf`).
     - Ensure the `datadir` path exists and MySQL has write permissions for this directory.

   - **Re-run Initialization After Clearing Data Directory:**
     - **Stop MySQL Server:**
       ```bash
       net stop MySQL
       ```
     - **Clear the Data Directory:**
       - Locate the data directory using the `datadir` setting in the MySQL configuration file.
       - Delete all files and folders within this directory.
     - **Re-run Initialization:**
       ```bash
       mysqld --initialize
       net start MySQL
       ```